pkg/gather/clusterconfig/pod_network_connectivity_checks.go

```diff
@@ -0,0 +1,85 @@
+package clusterconfig
+
+import (
+	"context"
+	"encoding/json"
+
+	controlplanev1 "github.com/openshift/api/operatorcontrolplane/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+// GatherPNCC collects a summary of failed PodNetworkConnectivityChecks.
+// The checks are requested via a dynamic client and
+// then unmarshaled into the appropriate structure.
+//
+// Resource API: podnetworkconnectivitychecks.controlplane.operator.openshift.io/v1alpha1
+// Docs for relevant types: https://pkg.go.dev/github.com/openshift/api/operatorcontrolplane/v1alpha1
+//
+// * Location in archive: config/podnetworkconnectivitychecks.json
+// * Id in config: pod_network_connectivity_checks
```

Comment:
You will need a bugzilla, but I am wondering if we can already add the version `4.8+`

Reply:
I guess it's okay to add it there if we can make it in time for 4.8.0.